src/coreclr/vm/nativelibrary.cpp

```diff
@@ -53,6 +53,7 @@ namespace
             LIMITED_METHOD_CONTRACT;
             m_hr = E_FAIL;
             m_priorityOfLastError = 0;
+            m_message = SString(SString::Utf8, "");
```

Comment:
You can simplify this to `m_message = SString::Empty()` or delete it completely and just depend on the default constructor. The default constructor will initialize the string to empty string.

Reply:
Thank you for note. Let me fix it.